osu.Game/Screens/Play/Player.cs

```diff
@@ -166,12 +166,8 @@ private void load(AudioManager audio, APIAccess api, OsuConfigManager config)
                     OnRetry = Restart,
                     OnQuit = Exit,
                     CheckCanPause = () => AllowPause && ValidForResume && !HasFailed && !RulesetContainer.HasReplayLoaded,
-                    OnPause = () =>
-                    {
-                        pauseContainer.Retries = RestartCount;
-                        hudOverlay.KeyCounter.IsCounting = !pauseContainer.IsPaused;
-                    },
-                    OnResume = () => hudOverlay.KeyCounter.IsCounting = true,
+                    OnPause = () => { RulesetContainer.KeyBindingInputManager.UseParentInput = false; },
+                    OnResume = () => {RulesetContainer.KeyBindingInputManager.UseParentInput = true; },
```

Comment:
`() => {Rul` typo

Reply:
this PR is a wip and this code is not final.